modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -269,6 +280,39 @@ public FluoConfiguration setWorkerThreads(int numThreads) {
   public int getWorkerThreads() {
     return getInt(WORKER_NUM_THREADS_PROP, WORKER_NUM_THREADS_DEFAULT);
   }
+  
+  public ObserverConfiguration getObserverConfig(String key) {
```

Comment:
Good idea.  I made the get method only return one observer as it fit better with the existing implementation of `FluoAdminImpl`.  However, I think refactoring `FluoAdminImpl` to use a get method that returns a `List` will help readability.  I'll change this.